Bump version to 0.6.0 (#195)
cybozu-go/moco: CHANGELOG.md

## [Unreleased]

## [0.6.0] - 2021-02-16

**Caution**

- Since MOCO v0.6.0, mysql data volumes (PVC) will be automatically deleted when the parent MySQLCluster resource is deleted.

>   If you want to keep the volumes, please edit the volumes (PVCs) manifests and delete their owner references manually.

Reviewer: ```suggestion
  If you want to keep the volumes, please  delete the owner reference manually from the volumes (PVCs) resource.
```
